src/FunderMaps.Webservice/Startup.cs

```diff
@@ -53,6 +58,9 @@ public void ConfigureServices(IServiceCollection services)
                 options.EnableForHttps = true;
             });
 
+            // Configure exception mapping.
+            services.AddSingleton<IExceptionMapper<FunderMapsCoreException>, FunderMapsExceptionMapper>();
```

Comment:
Extension?

Reply:
Already built that, forgot to add to startup.